### PR TITLE
Update ee

### PR DESCRIPTION
**Pull request by dalDevelo**
Updates ee.co.uk to include 2FA support via SMS, documentation + notes that this is a 1 way operation.

Screenshot 2024-09-03

### CODE REVIEW

File: entries/e/ee.co.uk.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
   "EE": {
     "domain": "ee.co.uk",
-    "contact": {
-      "facebook": "ee",
-      "twitter": "EE"
-    },
+    "url": "https://www.ee.co.uk/",
```
Comment (Carlgo11) [inline entries/e/ee.co.uk.json]: Thanks for the pull request @dalDevelo!
Could you please revert this change?
I'm not sure why the build failed earlier, but it seems to have been due to some random unrelated event, as the URL element shouldn't affect the domain rank.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by dalDevelo** [comment]
Done :)

Not sure why this happened either, but it's not the first time. I have had previous commits that failed for same / similar issues and I added URL, then removed it and it passed the 2nd time without it, as it has this time too. Might be something that needs looking at? 🤷🏻‍♂️ I was also confused as when I clicked on the test link the results displayed without issue on the output page.

Anyway, good to know the URL shouldn't be included for future ref, even if I add it to pass the test, as per above.

There are quite a few entries with the URL included, presumably from a time when this was required? If these can be removed I will happily submit updates and PR's for some of those too.

**Review by Carlgo11 — APPROVED** [review]

**Comment by hkamran80** [comment]
> There are quite a few entries with the URL included, presumably from a time when this was required? If these can be removed I will happily submit updates and PR's for some of those too.

We currently require the URL if going to `https://[domain]` fails. Feel free to submit PRs to update those that don't need it!

**Comment by Carlgo11** [comment]
What hkamran80 ^

I clearly type too slow
